Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -51,12 +51,19 @@ def runSelected(opt):
                      8, #BH/Cosmic MC
                      25, #MC ttbar
                      4.22, #cosmic data
+                     4.53, #run1 data + miniAOD
                      1000, #data+prompt
                      1001, #data+express
                      136.731, #2016B Photon data
+                     136.7611, #2016E JetHT reMINIAOD from 80X legacy
                      140.53, #2011 HI data
                      1330, #Run2 MC Zmm
-                     135.4 #Run 2 Zee ttbar
+                     135.4, #Run 2 Zee ttbar
+                     10042.0, #2017 ZMM
+                     10024.0, #2017 ttbar
+                     10824.0, #2018 ttbar
```

Comment:
workflows beyond 2017 do not exist in 80X, remove this line and the following two lines

Reply:
Thx, done. Please check.